docs/misc/deprecated.md

```diff
@@ -15,6 +15,10 @@ The following list of features are deprecated.
 ### Old Command Line Options
 **Deprecated In Release: [v1.8.0](/release-notes/#docker-engine-1-8-0)**
 
+The flags `-d` and `--daemon` are deprecated in favor of the `daemon` subcommand:
+
+    docker daemon -H ...
+
```

Comment:
When updating Docker, are init scripts (systemd, SysVinit) automatically updated? If not, should we add a mention/instructions that users should update them?